language-server/modules/langserver-core/src/test/resources/lspackageloader/user-home/repositories/local/bala/lstest/ls_local1/0.1.0/any/dependency-graph.json

```diff
@@ -0,0 +1,109 @@
+{
+  "packages": [
+    {
+      "org": "ballerina",
+      "name": "jballerina.java",
+      "version": "0.0.0",
+      "transitive": false,
+      "dependencies": [],
+      "modules": []
+    },
+    {
+      "org": "lstest",
+      "name": "ls_local1",
+      "version": "0.1.0",
+      "transitive": false,
+      "dependencies": [
+        {
+          "org": "ballerina",
+          "name": "io",
+          "version": "1.6.0",
+          "transitive": false,
+          "dependencies": [],
+          "modules": []
+        }
+      ],
+      "modules": []
+    },
+    {
+      "org": "ballerina",
+      "name": "io",
+      "version": "1.6.0",
+      "transitive": false,
+      "dependencies": [
+        {
+          "org": "ballerina",
+          "name": "jballerina.java",
+          "version": "0.0.0",
+          "transitive": false,
+          "dependencies": [],
+          "modules": []
+        },
+        {
+          "org": "ballerina",
+          "name": "lang.value",
+          "version": "0.0.0",
+          "transitive": false,
+          "dependencies": [],
+          "modules": []
+        }
+      ],
+      "modules": []
+    },
+    {
+      "org": "ballerina",
+      "name": "lang.value",
+      "version": "0.0.0",
+      "transitive": false,
+      "dependencies": [
+        {
+          "org": "ballerina",
+          "name": "jballerina.java",
+          "version": "0.0.0",
+          "transitive": false,
+          "dependencies": [],
+          "modules": []
+        }
+      ],
+      "modules": []
+    }
+  ],
+  "modules": [
+    {
+      "org": "lstest",
+      "package_name": "ls_local1",
+      "version": "0.1.0",
+      "module_name": "ls_local1",
+      "dependencies": [
+        {
+          "org": "ballerina",
+          "package_name": "io",
+          "version": "1.6.0",
+          "module_name": "io",
+          "dependencies": []
+        }
+      ]
+    },
+    {
+      "org": "lstest",
+      "package_name": "ls_local1",
+      "version": "0.1.0",
+      "module_name": "ls_local1.module2",
+      "dependencies": []
+    },
+    {
+      "org": "lstest",
+      "package_name": "ls_local1",
+      "version": "0.1.0",
+      "module_name": "ls_local1.module1.mod1",
+      "dependencies": []
+    },
+    {
+      "org": "lstest",
+      "package_name": "ls_local1",
+      "version": "0.1.0",
+      "module_name": "ls_local1.module1",
+      "dependencies": []
+    }
+  ]
+}
```

Comment:
There are some cases like this that require a newline at the bottom.

Reply:
These are generated(bala) files in .ballerina. We can consider those as ballerina archive files. Hense, adding newlines is not required as it is not a source code.